test/e2e/apimachinery/table_conversion.go

```diff
@@ -30,13 +30,20 @@ import (
 	metav1alpha1 "k8s.io/apimachinery/pkg/apis/meta/v1alpha1"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/kubernetes/pkg/printers"
+	utilversion "k8s.io/kubernetes/pkg/util/version"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
```

Comment:
Make sure the current versions under test compare to this correctly. Might need to make this v1.10.0-alpha.0

Reply:
@liggitt 
> Make sure the current versions under test (e.g. v1.10.0-beta.3) compare to this correctly. Might need to make this v1.10.0-alpha.0

Thanks, wrote a quick check to test cases like this locally. Updated the version here to `v1.10.0-alpha.0`.